main.py

```diff
@@ -1,10 +1,41 @@
-import init_django_orm  # noqa: F401
+import json
+from tkinter.font import names
 
-from db.models import Race, Skill, Player, Guild
+import init_django_orm  # noqa: F401
+import json
```

Comment:
The import statement for `json` is duplicated. You can remove one of them to clean up the code.